### DISCUSSION TITLE
Disable force update completely, because of 12v battery drain

### DISCUSSION THREAD

**Discussion by StanAnsems** [Hyundai-Kia-Connect/kia_uvo#844] [Ideas]
I had some issues with the 12v battery draining. I had the refresh interval set to 240 min but that seems to still drain the battery (not 100% sure if it is because of this though) I now changed it to 1440 min and hope it will not drain the battery anymore :crossed_fingers: 

It is not really clear to me why it would be needed/wanted it in the first place?
It would be nice to be able to completely disable the force update at all (to rule out the battery drain issue)

**Comment by erelke** [comment]
Change your battery.

**Comment by Esprit1st** [comment]
I don't think there is a legitimate reason to do a recurring force update. The days is the last update which doesn't change when parked. If the car is on or charging it update more frequently anyways. So I don't have an automatic force update and risk putting strain on the 12V. If needed I refresh manually.

**Reply by StanAnsems** [reply]
How do you prevent the automatic force update? setting the timer to an high number? or some other way?

**Reply by cdnninja** [reply]
Currently no good way to fully disable it.  Set the time to max.  It only calls a force update if data hasn't update in that many minutes.  So if a cached update has current data it won't use forced data.

**Comment by SteveDinn** [comment]
If the car is on, is there still danger of doing too many forced updates?  My (2025) car doesn't seem to update in a timely fashion after being started, it only seems to update reliably when being turned off, so sometimes it can look like it was never started for the whole day!

I'm planning to have an automation to detect if a phone (using the mobile_app integration) is both using android auto and connected to the car via bluetooth for 1 minute, and then force an update. I would like it to update more frequently when driving as well, which is why I'm asking if there is any detriment to doing forced updates at regular intervals while the car is on.

**Reply by profblackjack** [reply]
IMO it should be fine to trigger updates while the car is on. The root of the problem is around the fact that, if the car _isn't_ on, triggering an update wakes up all the cars electronics so they can report on sensors. This is a large load on the 12v battery, and the car's design around keeping the 12v battery topped off while parked is flawed for this use case. 
If it has to recharge the battery too many times in a row while the car isn't running, it enters a "battery must be bad" state and stops trying, which lets the 12v drain completely from the frequent update calls, and fully discharging the 12v is very bad for the battery's health.

**Reply by SteveDinn** [reply]
Thanks for confirming. This is what I figured.
